wizard/src/utils/detect_place.ts

```diff
@@ -16,23 +16,26 @@
 import { DetectedDetails, DetectedFormat, ConfidenceLevel } from "../types";
 import countriesJSON from "./country_mappings.json";
 
-// A PlaceDetector objected is meant to be initialized once. It provides
-// convenience access to all place types and their supported formats. It also
-// supports detecting the place type for each individual column (a header and
-// a list of string values).
+/**
+* A PlaceDetector objected is meant to be initialized once. It provides
+* convenience access to all place types and their supported formats. It also
+* supports detecting the place type for each individual column (a header and
+* a list of string values).
+*/
 export class PlaceDetector {
   countryNames: Set<string>;
   countryISO: Set<string>;
   countryAbbrv3: Set<string>;
   countryNumeric: Set<string>;
 
-  placeTypes: Set<string>;
+  placeTypes: Map<string, string>;
 
   static typeFormatMappings = new Map<string, Array<DetectedFormat>>(
     [
-      ["Longitude",   [ {propertyName: "name", displayName: "Full Name"},]],
-      ["Latitude",    [ {propertyName: "name", displayName: "Full Name"},]],
-      ["LatLon",      [ {propertyName: "name", displayName: "Full Name"},]],
+      ["Longitude",   [ {propertyName: "longitude", displayName: "Longitude"},]],
+      ["Latitude",    [ {propertyName: "latitude", displayName: "Latitude"},]],
+      ["LatLon",      [ {propertyName: "GeoCoordinates", displayName: "Geo Coordinates"},]],
```

Comment:
Two things:

1. propertyName "GeoCoordinates" doesn't seem right?
2. Do we need entries for "Latitude", "Longitude" and "LatLon" ?  Rather should lat/lon etc be properties of "GeoCoordinates" entry?

Reply:
Let me follow up with you offline on this. I'll merge this one and make the changes needed in the follow-up PR(s).

Reply:
Sure!